examples/LanguageIdMediaUrl.java

```diff
@@ -0,0 +1,95 @@
+        LanguageIdClient languageIdClient = new LanguageIdClient(accessToken);
+
+        // Set up source configuration parameters
+        String mediaUrl = "https://www.rev.ai/FTC_Sample_1.mp3";
+        // Authorization header is optional; use it if needed to access the source media url
+        Map<String, String> sourceAuth = new HashMap<>();
+        sourceAuth.put("Authorization", "Bearer <source_token>");
+
+        // Set up notification configuration parameters
+        String callbackUrl = "https://example.com";
+        // Authorization header is optional; use it if needed to access the callback url
+        Map<String, String> notificationAuth = new HashMap<>();
+        notificationAuth.put("Authorization", "Bearer <notification_token>");
+
+        // Initialize the LanguageIdJobOptions object and assign
+        LanguageIdJobOptions languageIdJobOptions = new LanguageIdJobOptions();
+        languageIdJobOptions.setSourceConfig(mediaUrl, sourceAuth);
+        languageIdJobOptions.setMetadata("My first submission");
+        languageIdJobOptions.setNotificationConfig(callbackUrl, notificationAuth);
+        languageIdJobOptions.setDeleteAfterSeconds(2592000); // 30 days in seconds
+
+        LanguageIdJob submittedJob;
+
+        try {
+            // Submit job with language id options
+            submittedJob = languageIdClient.submitJobUrl(languageIdJobOptions);
+        } catch (IOException e) {
+            throw new RuntimeException("Failed to submit url [" + mediaUrl + "] " + e.getMessage());
+        }
+        String jobId = submittedJob.getJobId();
+        System.out.println("Job Id: " + jobId);
+        System.out.println("Job Status: " + submittedJob.getJobStatus());
+        System.out.println("Created On: " + submittedJob.getCreatedOn());
+
+        // Waits 5 seconds between each status check to see if job is complete
+        boolean isJobComplete = false;
+        while (!isJobComplete) {
+            LanguageIdJob retrievedJob;
+            try {
+                retrievedJob = languageIdClient.getJobDetails(jobId);
+            } catch (IOException e) {
+                throw new RuntimeException("Failed to retrieve job [" + jobId + "] " + e.getMessage());
+            }
+
+            LanguageIdJobStatus retrievedJobStatus = retrievedJob.getJobStatus();
+            if (retrievedJobStatus == LanguageIdJobStatus.COMPLETED
+                    || retrievedJobStatus == LanguageIdJobStatus.FAILED) {
+                isJobComplete = true;
+            } else {
+                try {
+                    Thread.sleep(5000);
+                } catch (InterruptedException e) {
+                    e.printStackTrace();
+                }
+            }
+        }
+
+        // Get the language id result
+        LanguageIdResult languageIdResult;
+
+        try {
+            languageIdResult = languageIdClient.getResultObject(jobId);
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
+
```

Comment:
Do you think its worth having some example of how they would take a languageIdResult and use it to submit a transcription job with that language? or is that too simple

Reply:
hmmmmmm I think it's pretty simple but I guess it doesn't mean we can't add it.